src/components/organizations/OrganizationShow.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  Show,
+  TabbedShowLayout,
+  Tab,
+  TextField,
+  ReferenceManyField,
+  SimpleList,
+  TopToolbar,
+  DeleteButton,
+} from "react-admin";
+
+const useStyles = makeStyles({
+  root: {
+    color: "#3f51b5",
+  },
+});
+
+const OrganizationShowActions = ({ basePath, data, resource }) => (
+  <TopToolbar>
+    <DeleteButton basePath={basePath} record={data} resource={resource} />
+  </TopToolbar>
+);
+
+const OrganizationShow = (props) => {
+  const classes = useStyles();
+
+  return (
+    <Show label="Show" actions={<OrganizationShowActions />} {...props}>
+      <TabbedShowLayout>
+        <Tab label="details">
+          <TextField label="ID" source="organizationId" />
+          <TextField source="organizationName" />
+        </Tab>
+        <Tab label="applications">
+          <ReferenceManyField
+            link="show"
+            label=""
+            reference="Applications"
```

Comment:
this can reference `Organizations` directly so that we can use the endpoint made for this directly `GET /msadmins/organizations/:organizationId/applications`

Reply:
![image](https://user-images.githubusercontent.com/17856665/88447097-96f66d00-ce27-11ea-9a3b-26f2395a9dd4.png)
![image](https://user-images.githubusercontent.com/17856665/88447101-9b228a80-ce27-11ea-9935-673317d57a21.png)
![image](https://user-images.githubusercontent.com/17856665/88447105-9eb61180-ce27-11ea-9f62-3a7fe15cad75.png)
![image](https://user-images.githubusercontent.com/17856665/88447108-a2499880-ce27-11ea-9aca-640b216d75cf.png)